pom.xml

```diff
@@ -180,9 +180,8 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.20.1</version>
+                <version>3.0.0-M1</version>
```

Comment:
this is milestone version,  is there plan on official release?

Reply:
They have not made any commitment yet. I will follow up the official release.